yarn.lock

```diff
@@ -13001,19 +13052,19 @@ typescript@3.2.x:
   integrity sha512-0RNDbSdEokBeEAkgNbxJ+BLwSManFy9TeXz8uW+48j/xhEXv1ePME60olyzw2XzUqUBNAYFeJadIqAgNqIACwg==
 
 typescript@^3.2.4:
-  version "3.3.4000"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.3.4000.tgz#76b0f89cfdbf97827e1112d64f283f1151d6adf0"
-  integrity sha512-jjOcCZvpkl2+z7JFn0yBOoLQyLoIkNZAs/fYJkUG6VKy6zLPHJGfQJYFHzibB6GJaF/8QrcECtlQ5cpvRHSMEA==
+  version "3.4.1"
```

Comment:
Was it intentional that we didn't specify the typescript dependency with `~`?

Reply:
I guess so, yeah. Why shouldn't we be willing to go along with the latest patch version of TypeScript in this project? Also at the risk that a lockfile update will fail.

Reply:
I will merge this PR despite your comment. If necessary, we can adjust the TypeScript version declaration in a separate commit if we decide so.